src/components/InputGlobal/index.tsx

```diff
@@ -18,6 +18,7 @@ const Input: React.FC<InputGlobal> = ({
 }) => {
   const [search, setSearch] = useState('');
   const [error, setError] = useState(false);
+  const [linkPage, setLinkPage] = useState('');
```

Comment:
is there another user action that needs to be monitored besides the text input or mouse focus in/out of the pop up? if not, just create a variable that stores the data you want to save